File: converter/src/main/java/com/example/converter/infra/config/WebConfig.java
package com.example.converter.infra.config;

import com.example.converter.presentataion.converter.LocalDateConvereter;
import com.example.converter.presentataion.formmater.LocalDateTimeFormatter;
import org.springframework.context.annotation.Configuration;
import org.springframework.format.FormatterRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

@Configuration
public class WebConfig implements WebMvcConfigurer {
    @Override
    public void addFormatters(FormatterRegistry registry) {
        registry.addConverter(new LocalDateConvereter());
        registry.addFormatter(new LocalDateTimeFormatter());
    }
}

File: feign-client-example/src/main/kotlin/org/example/feignclientexample/config/WebConfig.kt
package org.example.feignclientexample.config

import com.fasterxml.jackson.databind.ObjectMapper
import org.example.feignclientexample.constants.ObjectMapperConstants
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Primary

@Configuration
class WebConfig {
    @Bean
    @Primary
    fun objectMapper(): ObjectMapper = ObjectMapperConstants.DEFAULT_OBJECT_MAPPER
}